README.md

```diff
@@ -49,90 +37,84 @@ app.use(errorHandler({
     "strong-error-handler": {
       "params": {
          "debug": false,
-         "log": true,
+         "log": true
        }
     }
   }
 }
 ```
 
-## Content Type
+The above configuration will log errors to the server console, but not return stack traces in HTTP responses.
+For details on configuration options, see below.
 
-Depending on the request header's `Accepts`, response will be returned in
- the corresponding content-type, current supported types include:
-- JSON (`json`/`application/json`)
-- HTML (`html`/`text/html`)
+### Response format and content type
 
-*There are plans to support other formats such as Text and XML.*
+The `strong-error-handler` package supports HTML and JSON responses:
 
-## Options
-
-#### debug
-
-`boolean`, defaults to `false`.
+- When the object is a standard Error object, it returns the string provided by the stack property in HTML/text
+  responses.
+- When the object is a non-Error object, it returns the result of `util.inspect` in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties from the object in the response.
 
-If you need to set the environment to development mode, you will need to change the value to `true`.
+The content type of the response depends on the request's `Accepts` header.
 
-When enabled, HTTP responses include all error properties, including
-sensitive data such as file paths, URLs and stack traces. *See Examples below*
+-  For Accepts header `json` or `application/json`, the response content type is JSON.
+-  For Accepts header `html` or `text/html`, the response content type is HTML.
 
-#### log
+*There are plans to support other formats such as Text and XML.*
 
-`boolean`, defaults to `true`.
+## Options
 
-When enabled, all errors are printed via `console.error`. That includes an array of fields (custom error properties)
- that are safe to include in response messages (both 4xx and 5xx).
+| Option | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| debug | Boolean&nbsp;&nbsp;&nbsp; | `false` | If `true`, HTTP responses include all error properties, including sensitive data such as file paths, URLs and stack traces. *See Examples below* |
+| log | Boolean | `true` |  If `true`, all errors are printed via `console.error`, including an array of fields (custom error properties) that are safe to include in response messages (both 4xx and 5xx). <br/> If `false`, sends only the error back in the response. |
 
-When not enabled, it only sends the error back in the response.
+### Customizing log format
 
-Customization of the log format is intentionally not allowed. If you would like
-to use a different format/logger, disable this option and add your own custom
-error-handling middleware.
+To use a different log format, disable `errorHandler.log` and add your own custom
+error-handling middleware; for example:
 
 ```js
-app.use(myErrorLogger());
 app.use(errorHandler({ log: false }));
+app.use(myErrorLogger());
```

Comment:
> If it MUST be in the opposite order then we should state so explicitly and also reverse the clauses in the prior sentence

Yes please, reverse the order in the text.

> I did not test it, but I have been unsuccessful in getting it to run in either order ....which begs the question: where does this code go? And are there any requirements on the myErrorLogger function? Must it conform to the standard of Express error middleware? Or...?

The example code is for an Express app. Since there is no standard convention for structuring Express server file (AFAIK), it's hard to tell where exactly this code should go. In general, strong-error-handler should be the last middleware added to the server, i.e. it should be just before `app.listen` call.

```js
var app = express();
var errorHandler = require('strong-error-handler');

// add request pre-processing middleware
// register your routes

// log errors
app.use(function myErrorLogger(err, req, res, next) {
  console.log('unhandled error' ,err);
  next(err);
});

// the ultimate error handler
app.use(errorHandler({ log: false }));

app.listen(3000, function() {
  console.log('Server listening at http://localhost:3000/');
});
```

> Not sure whats the best way to document how to use it for LB app 

For LoopBack applications, the user need to put the custom error-logging middleware into a standalone file. I was writing an example recently, I wish I knew where it was :( Let me type it again:

```
// server/middleware/error-logger.js
module.exports = function(options) {
  return function logError(err, req, res, next) {
    console.log('unhandled error' ,err);
    next(err);
  };
};

// server/middleware.json
{
  // ...
  "final:after": {
    "./middleware/error-logger": {},
    "strong-error-handler": {
      "params": {
        log: false
      }
    }
}
```

Note that the default `middleware.development.json` file explicitly enables logging in strong-error-handler params, so users should probably change that file too.